test/persistence-hooks.suite.js

```diff
@@ -736,10 +739,18 @@ module.exports = function(dataSource, should) {
           { id: existingInstance.id, name: 'updated name' },
           function(err, instance) {
             if (err) return done(err);
-            observedContexts.should.eql(aTestModelCtx({
-              where: { id: existingInstance.id },
-              data: { id: existingInstance.id, name: 'updated name' }
-            }));
+            if (dataSource.connector.updateOrCreate) {
```

Comment:
Please add a comment explaining why we get different behaviour depending on the connector implementation.